### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by barucoh**

### CODE REVIEW

File: opensource_case/network/network_topolgy.yaml [hunk 1]
```diff
@@ -20,29 +20,34 @@ inputs:
...
 
   private_subnet_application_pool:
+    description: IP pool for the private network to associate IP from.
     default:
     - start: 10.10.1.2
       end: 10.10.1.254
 
   mgmt_network_id:
-    description: manager network
+    description: Cloudify Manager's network
```
Comment (EarthmanT) [inline opensource_case/network/network_topolgy.yaml]: Is this the ID of a network in Openstack?

File: opensource_case/network/network_topolgy.yaml [hunk 2]
```diff
@@ -20,29 +20,34 @@ inputs:
...
 
   router_id:
-    description: manager network
+    description: Cloudify Manager's connected router
```
Comment (EarthmanT) [inline opensource_case/network/network_topolgy.yaml]: same

**Review by EarthmanT — CHANGES_REQUESTED** [review]

**Review by EarthmanT — APPROVED** [review]

File: opensource_case/network/network_topolgy.yaml (hunk 1 above)
Comment (barucoh) [reply opensource_case/network/network_topolgy.yaml]: Yes, I'll fix the description

File: opensource_case/network/network_topolgy.yaml (hunk 2 above)
Comment (barucoh) [reply opensource_case/network/network_topolgy.yaml]: Fixed

**Pull request merged**